content/en/docs/Features/validations.md

```diff
@@ -962,11 +962,11 @@ Fix the parentRefs field to target to an existing gateway.
 
 ### KIA1402 - BackendRef on rule doesn't have a valid service (Service name not found)
 
-Gateway API HTTPRoute could be pointing to a Service inside your mesh the Route sends the traffic to. A Service name should be specified, not a hostname. This Service can be a certain version of a parent Service, but in that case a separate Service is required to be created. When the namespace field is not specified it takes Service from the current HTTPRoute's namespace. Here the error indicates that the referenced Service is not found in the provided namespace.
+Gateway API HTTPRoute could be pointing to a Service inside your mesh the Route sends the traffic to. A Service name should be specified, not a hostname. This Service can be a certain version of a parent Service, but in that case a separate Service is required to be created. When the namespace field is not specified it takes Service from the current HTTPRoute's namespace. In a case of referencing to a Service from remote namespace, a ReferenceGrant object needs to be created to enable cross namespace references.  Here the error indicates that the referenced Service is not found in the provided namespace or the ReferenceGrant is missing (in a case of remote namespace).
 
 #### Resolution
 
-Correct the backendRefs name to point to a correct Service (in this namespace or to other namespaces), deploy the missing Service to the mesh or remove the configuration linking to that non-existing Service.
+Correct the backendRefs name to point to a correct Service (in this namespace or to other namespaces), deploy the missing Service to the mesh, create a ReferenceGrant object in a case of remote namespace or remove the configuration linking to that non-existing Service.
```

Comment:
It is not clear to me which of the actions should be taken. Maybe: 

To correct the backendRefs name to point to a correct Service (in this namespace or to other namespaces):
- Deploy the missing Service to the mesh and create a ReferenceGrant object in a case of a remote namespace
- Or remove the configuration linking to that non-existing Service.

Reply:
right, restructured